iceoryx_posh/source/roudi/roudi.cpp

```diff
@@ -212,13 +213,17 @@ void RouDi::processMessage(const runtime::IpcMessage& message,
         }
         else
         {
-            uint32_t pid;
-            uid_t userId;
-            int64_t transmissionTimestamp;
+            uint32_t pid{0};
```

Comment:
Please write `uint32_t pid{0U};`

Sadly there is only a clang-tidy check for `0u` -> `0U` but not when the `U` is missing at all. Therefor we have to write a custom clang-tidy rule.

Reply:
@elfenpiff That's a good idea, happy to write another custom clang tidy rule :) I wrote one which adds `noexcept` to all methods. First version is available [here](https://github.com/mossmaurice/llvm-project/tree/add-custom-noexcept-all-the-things-rule).

@r7vme Could you give it a try and think about we can integrate this in our iceoryx workflow? Probably we should fork the llvm repo on the ApexAI organization.

cc @jpsamper2009

Reply:
@mossmaurice This is addressed here: https://github.com/eclipse-iceoryx/iceoryx/issues/902